configure_template.R

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env Rscript
+library(git2r)
+library(stringr)
+library(readr)
+path <- file.path(tempfile(pattern = "git2r-"), "template")
+dir.create(path, recursive = TRUE)
+repo <- clone("https://github.com/rstudio/template", branch = "radix", path)
+system2("npm", args = c("--prefix", path, "install"))
+system2("npm", args = c("--prefix", path, "run", "build"))
+read_lines(file.path(path, "dist", "template.v2.js")) %>%
+  str_replace(fixed("/[a-z0-9_]+(?=\\()/i"), "/[a-z\\.0-9_]+(?=\\()/i") %>%
```

Comment:
Do you feel confident that these regexes will only apply the changes we did before? (i.e. could they leak to other parts of the script). If you are a bit uncertain is there a way to scope the change down to make it safer?

Reply:
LMK what you think about the regexes (save for that it looks like we are ready to merge).